### DISCUSSION TITLE
Random::otp returns string instead of int

### DISCUSSION THREAD

**Discussion by binaryfire** [valorin/random#10] [Q&A] [answered]
Hey @valorin,

Is there a reason why `Random::otp()` returns a string? Wouldn't it be better to return an int since it's numeric?

**Comment by valorin** [comment, marked as answer]
It needs to be a string so it keeps the leading zeros. An integer won't keep them, which is the whole point of the function.

Otherwise, you'll end up with less than 6 digits or be forced to start at something like `100000` instead of `000000`.

Good question though - I should make it more obvious.
